maintainers: education session (#39)
euroscipy/euroscipy.github.io: content/pages/2022/maintainers.md

infrastructure, and is now ready to engage more widely to help grow and support
the community.

#### Education - Materials, methods, tools
**Chair: Mx Chiin-Rui Tan (She/Her, They/Them)**
**Time: <TBA>**

This session focuses on issues related to education in the ecosystem, from
three different aspects, and during the session we focus on recent advances and
existing and upcoming challenges.

- Materials: how are projects dealing with documentation and education
  materials

> - Methods: What should we do to make our materials more accessible to
>   underrepresented groups? How can they be more accessible?

Reviewer: ```suggestion
- Methods: What should we do to make our materials more accessible to
  underrepresented and/or historically marginalised groups?
```
